oidc-controller/api/core/acapy/client.py

```diff
@@ -134,12 +118,13 @@ def oob_create_invitation(
         create_invitation_payload = {
             "attachments": [
                 {
-                    "id": presentation_exchange["presentation_exchange_id"],
+                    "id": presentation_exchange["pres_ex_id"],
                     "type": "present-proof",
                     "data": {"json": presentation_exchange},
                 }
             ],
             "use_public_did": use_public_did,
+            "trace": True,
```

Comment:
Is this flag required? I think we only want to add trace when troubleshooting, so it should not be set to `true` by default. Might want to have this removed, or follow the `DEBUGGER` environment value maybe.